packages/@react-aria/dnd/src/useDroppableCollection.ts

```diff
@@ -206,8 +206,8 @@ export function useDroppableCollection(props: DroppableCollectionOptions, state:
 
       let {keyboardDelegate} = localState.props;
       let nextKey = target.type === 'item'
-        ? keyboardDelegate.getKeyBelow(target.key)
-        : keyboardDelegate.getFirstKey();
+        ? keyboardDelegate.getKeyBelow(target.key, {allowsDisabled: true})
+        : keyboardDelegate.getFirstKey(null, null, {allowsDisabled: true});
```

Comment:
Feels a bit gross needing all the unused options haha. Did you try using the `state.collection` instead? It should also have similar `getFirstKey`, `getKeyBelow`, etc and doesn't skip disabled keys by default. 

I'm not sure if we need the other keyboard delegate getters like `getPageUp`/etc, but it seems like useDroppableCollection only uses above/below/first/last key getters.

Reply:
I had opted to use the keyboardDelegate first, because that's how we actually navigate, even though yeah, those empty null args feel gross
If I did it with state.collection, then I think i'd just be re-implementing keyboardDelegate
this is more obvious when you think about our grid layouts such as waterfall

Reply:
Ahhh, I forgot about CardView and how it will support drag and drop. Yeah, the collection route is a no go for that